databases/postgresql-activerecord.md

```diff
@@ -0,0 +1,139 @@
+# PostgreSQL
+
+[PostgreSQL](http://www.postgresql.org/) is an object-relational database management system (ORDBMS).
+
+The connection between PostgreSQL and Ruby happens via the [`pg`](https://rubygems.org/gems/pg) gem.
+
+## Installation
+
+[Install PostgreSQL](http://www.postgresql.org/download/)
+
+
+Install Ruby Postgres bridge
+```
+gem install pg
+```
+
+*Note:* OSX Users may have troubles with the installation of the `pg` gem. Read [this](http://stackoverflow.com/a/19850273/2980299)
+
+
+```
+gem install activerecord
+gem install sinatra-activerecord
+gem install rake # to apply migrations
+```
+
+
+## Connect to the database
+
+Either by making a `config/database.yml` file that gets automatically loaded
+
+```
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: database_name
+  pool: 2
+  username: your_username
+  password: your_password
+
+production:
+  adapter: postgresql
+  encoding: unicode
+  database: database_name
+  pool: 2
+  username: your_username
+  password: your_password
+```
+
+or by specifying in your sinatra app
+
+```ruby
+configure :development do
+  set :database, {adapter: 'postgres',  encoding: 'unicode', database: 'your_database_name', pool: 2, username: 'your_username', password: 'your_password'}
+end
+
+configure :production do
+  set :database, {adapter: 'postgres',  encoding: 'unicode', database: 'your_database_name', pool: 2, username: 'your_username', password: 'your_password'}
+end
+```
+
+## Create models
+```ruby
+class Article < ActiveRecord::Base
+end
+```
+
+## Create migrations
+
+First, create a `Rakefile` or update your `Rakefile` to require `activerecord/rake`
+
+```
+require 'sinatra/activerecord/rake'
+...
+```
+
+Then create a migration as shown:
+
+```
+▶ rake db:create_migration NAME=create_articles
+db/migrate/20150516144225_create_articles.rb
```

Comment:
This looks a bit ambiguous. It looks as if we're asking to type-in the `db/migrate/...` part too. Do you mind if I fix this?